packages/nook/src/components/LibraryView/LibraryView.tsx

```diff
@@ -1,5 +1,98 @@
+"use client";
+
+import React from "react";
+import { View, Text, Loader, Divider, Table, Badge, Alert } from "reshaped"
+import Markdown from 'react-markdown';
```

Comment:
I would try to avoid heavy client side dependencies here (40+kb). I think we can start without markdown at first and once we get to rendering docs - we can do this on the server/build side instead